src/libman/Contracts/Constants.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Web.LibraryManager.Tools.Contracts
+{
```

Comment:
We have these constants all over all extensions. We should move it to a common place. Maybe at the Microsoft.Web.LibraryManager project